docs/cli/csv-parser-order.md

```diff
@@ -0,0 +1,349 @@
+# CSV Parser Order
+
+[![Travis Build Status][travis-icon]][travis]
+[![Codecov Coverage Status][codecov-icon]][codecov]
+[![David Dependencies Status][david-icon]][david]
+[![David devDependencies Status][david-dev-icon]][david-dev]
+
+Convert [commercetools order](https://dev.commercetools.com/http-api-projects-orders.html) CSV data to JSON. See examples below for CSV format and sample responses.
+
+## Usage
+`npm install @commercetools/csv-parser-order --global`
+
+### CLI
+```
+Usage: csvparserorder [options]
+Convert commercetools order CSV data to JSON.
+
+Options:
+  --help, -h        Show help text.                                    [boolean]
+  --version, -v     Show version number.                               [boolean]
+  --type, -t        Predefined type of csv.
+                             [required] [choices: "lineitemstate", "returninfo", 'deliveries']
```

Comment:
Just incase, to generate it, run the help on your terminal, copy the output and paste here...